.github/workflows/PR-build.yml

```diff
@@ -24,116 +24,172 @@ env:
   PACKAGING_ROOT: build/packages 
 
 jobs:
+  changes:
```

Comment:
Why was paths-ignore not enough?

Reply:
Per @anuraaga up above:
> Generally this setting doesn't work since required checks won't be run on docs PRs and the PRs become unmergable.
> 
> I notice this repo doesn't have required checks but it probably should per best practice. I think there are other solutions that work on the step level that could work in conjunction with that.

If it's done on a paths-ignore level, then none of the jobs will run. This way, all of the jobs run, but they exit quickly. See [PR Build #805](https://github.com/aws-observability/aws-otel-collector/actions/runs/1423228261)